hummingbot/client/ui/hummingbot_cli.py

```diff
@@ -98,15 +101,22 @@ def __init__(self,
     def did_start_ui(self):
         self._stdout_redirect_context.enter_context(patch_stdout(log_field=self.log_field))
 
-        log_level = global_config_map.get("log_level").value
-        init_logging("hummingbot_logs.yml", override_log_level=log_level)
+        log_level = self.client_config_map.log_level
+        init_logging("hummingbot_logs.yml", self.client_config_map, override_log_level=log_level)
 
         self.trigger_event(HummingbotUIEvent.Start, self)
 
     async def run(self):
-        self.app = Application(layout=self.layout, full_screen=True, key_bindings=self.bindings, style=load_style(),
-                               mouse_support=True, clipboard=PyperclipClipboard())
+        self.app = Application(
+            layout=self.layout,
+            full_screen=True,
+            key_bindings=self.bindings,
+            style=load_style(self.client_config_map),
+            mouse_support=True,
+            clipboard=PyperclipClipboard(),
+        )
         await self.app.run_async(pre_run=self.did_start_ui)
+        # await self.app.run_async(pre_run=partial(self.did_start_ui, self.app))
```

Comment:
Is this required or should it be removed?

Reply:
Removed